index.html

```diff
@@ -0,0 +1,54 @@
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;500;600;700;800&family=Roboto:wght@300;400;500&display=swap" rel="stylesheet"> 
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css" integrity="sha512-xh6O/CkQoPOWDdYTDqeRdPCVd1SpvCA9XXcUnZS2FmJNp1coAFzvtCN9BmamE+4aHK8yyUHUSCcJHgXloTyT2A==" crossorigin="anonymous" referrerpolicy="no-referrer" />    <link rel="stylesheet" href="styles.css">
+    <script src="./script.js"></script>
+  </head>
+  <body>
+    <main>
```

Comment:
Great job @Kweeka1 🤝

- [ ] [Optional] Since this milestone is organized in two sections, you may remove the `<main>` tag, and change the `<nav>` into a `<header>` tag. This `<header>` tag will represent the first section and then the `<section>` tag will represent the second section. surround the hamburger icon with a `<nav>` tag instead of a `<div>`.

Reply:
In the first PR review. I got a comment saying that I should replace ```header``` tag with a ```nav``` tag.
>  Kindly make sure the logo and the Hmburger are wrapped inside a nav tag for accessibility and the semanric sake.

I'm really confused.
@1GORDON